runtime/src/iree/base/internal/threading_win32.c

```diff
@@ -322,6 +322,10 @@ void iree_thread_resume(iree_thread_t* thread) {
   IREE_TRACE_ZONE_END(z0);
 }
 
+void iree_thread_join(iree_thread_t* thread) {
+  WaitForSingleObject(thread->handle, INFINITE);
```

Comment:
add IREE_TRACE_ZONE_BEGIN/END around these implementations so we can easily see when a blocking join happens

Reply:
I added them to all implementations.